components/Tel.Egram.Components.Application/ApplicationController.cs

```diff
@@ -142,6 +175,31 @@ private void GoToWorkspacePage()
             Model.AuthenticationModel = null;
         }
 
+        private void UpdateConnectionState(int state)
+        {
+            if (Model.AuthenticationModel == null)
```

Comment:
I don't understand why AuthenticationModel is getting activated by this method. Is your goal just to update TextBlock's value?